lib/include/jxl/decode.h

```diff
@@ -833,10 +834,19 @@ JXL_EXPORT JxlDecoderStatus JxlDecoderGetColorAsICCProfile(
  * No matter what, the @ref JXL_COLOR_PROFILE_TARGET_DATA must still be queried
  * to know the actual data format of the decoded pixels after decoding.
  *
- * The JXL decoder has no color management system built in, but can convert XYB
- * color to any of the ones supported by JxlColorEncoding. Note that if the
- * requested color encoding has a narrower gamut, or the white points differ,
- * then the resulting image can have significant color distortion.
+ * The JXL decoder has no color management system (CMS) built in, but can
+ * convert XYB color to any of the ones supported by JxlColorEncoding. Note that
+ * if the requested color encoding has a narrower gamut, or the white points
+ * differ, then the resulting image can have significant color distortion.
+ *
+ * Can be used with a CMS by setting it via JxlDecoderSetCms, in which case it
```

Comment:
Should we require JxlDecoderSetCms to be called even if libjxl has a default cms available?
In the encode api we don't require that, if I understand correctly.

Reply:
Yes, good idea. I tried to make that clear in the latest commit.